src/Halogen/HTML/Core.purs

```diff
@@ -70,17 +70,18 @@ text :: forall p i. String -> HTML p i
 text = HTML <<< VDom.Text
 
 -- | A smart constructor for HTML elements.
-element :: forall p i. VDom.ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
-element = coe (\name props children -> VDom.Elem (VDom.ElemSpec Nothing name props) children)
+element :: forall p i. Maybe VDom.Namespace -> VDom.ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
+element ns =
+  coe (\name props children -> VDom.Elem (VDom.ElemSpec ns name props) children)
   where
```

Comment:
Yeah, I consider this the correct signature. It's just a matter of whether we consider this a breaking change. If it is, we might just have to copy it to a new definition and define the old one in terms of this.